CHANGELOG.md

```diff
@@ -17,6 +17,19 @@ available as `StandardHealthMonitorServiceActor`.
 
 ### Bug fixes
 - Fixed an issue that could cause Cromwell to consume disk space unnecessarily when using zipped dependencies
+- Fixed an issue that caused the `write_map()` function in Cromwell's WDL 1.0 implementation to produce output in the wrong format. Specifically, the output's rows and columns were swapped. WDL draft-2 was not affected.
```

Comment:
This changelog entry should be added to a new `## 39 Release Notes` section. Also add a `### Bug fixes` along with a `####` descriptive header for this specific bug fix as well.

Reply:
Good catch and not the first time I've done this!